src/js/containers/review/ReviewDataContainer.jsx

```diff
@@ -8,7 +8,6 @@ import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 
 import * as ReviewHelper from '../../helpers/reviewHelper.js';
-import * as AgencyHelper from '../../helpers/agencyHelper.js';
```

Comment:
Removing this here removes the Agency Name info from the Review & Certify page correct?